Project.toml

```
@@ -5,13 +5,15 @@ version = "0.10.4"
 [deps]
 Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
+NLPModelsModifiers = "e01155f1-5c6f-4375-a9d8-616dd036575f"
 SolverCore = "ff4d7338-4cf1-434d-91df-b86cb86fb843"
 
 [compat]
 Ipopt = "1"
 NLPModels = "0.14, 0.15, 0.16, 0.17, 0.18, 0.19, 0.20, 0.21"
```

```suggestion
NLPModels = "0.19, 0.20, 0.21"
```
the others are not compatible with NLPModelsModifiers 0.7